ofrak_core/ofrak/core/gzip.py

```diff
@@ -87,19 +82,48 @@ class GzipPacker(Packer[None]):
     """
 
     targets = (GzipData,)
-    external_dependencies = (PIGZ,)
```

Comment:
So basically this will run even if PIGZ isn't installed; OFRAK does not "track" this third party dependency.

This seems like an OK tradeoff; we can put it in our Docker image, and people can use this even if it is not installed.

(Writing this mainly to document this for posterity).